### PR TITLE
added Amharic language

### PR DESCRIPTION
**Pull request by Hen0k**
Review this @casey-collab

### CODE REVIEW

File: language.md [hunk 1]
```diff
@@ -167,3 +167,5 @@ Dreams to Memories – Wisdom in Ladakh.To read this click [here](Himalayan_Odys
 Astoli Dothraki ? [Dothrae jinne](dothraki/dothraki_evoli.md)
 
 Si Latine legere vis, [hic preme](latin/arena.md)
+
+አማርኟ ኸቻልክ [ተጫን](amharic/ጀምር.md)
```
Comment (casey-collab) [inline language.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Amharic/ጀምር.md [hunk 2]
```diff
@@ -0,0 +1 @@
+[ጀምር.md](ሀበሻ/፩.md)
```
Comment (casey-collab) [inline Amharic/ጀምር.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Hen0k** [comment]
thanks @casey-collab

**Comment by casey-collab** [comment]
Looks great! :penguin:

**Pull request merged**